lib/rubocop/rspec/cop_helper.rb

```diff
@@ -46,12 +46,21 @@ def autocorrect_source(source, file = nil)
     @last_corrector.rewrite
   end
 
-  def _investigate(cop, processed_source)
-    team = RuboCop::Cop::Team.new([cop], nil, raise_error: true)
+  def _investigate(cops, processed_source)
+    cop_array = Array(cops)
+    team = RuboCop::Cop::Team.new(cop_array, nil, raise_error: true)
     report = team.investigate(processed_source)
-    @last_corrector = report.correctors.first || RuboCop::Cop::Corrector.new(processed_source)
```

Comment:
I'm not sure why there's a `first` there. All correctors should be merged, I think `correctors.compact.inject(:merge!)` should work, and that method should be added to `InvestigationReport`

Reply:
Not sure I 100% understand how you'd like this to look, but I'll give it a go and you'll let me know if I did it wrong. 😛